sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Generated/Models/UpdateModelSettingsRequest.cs

```diff
@@ -5,13 +5,13 @@
 
 #nullable disable
 
-namespace Azure.Iot.TimeSeriesInsights
+namespace Azure.Iot.TimeSeriesInsights.Models
 {
     /// <summary> Request to update model settings. One of &quot;name&quot; or &quot;defaultTypeId&quot; must be set. </summary>
-    public partial class UpdateModelSettingsOptions
+    public partial class UpdateModelSettingsRequest
     {
-        /// <summary> Initializes a new instance of UpdateModelSettingsOptions. </summary>
-        public UpdateModelSettingsOptions()
+        /// <summary> Initializes a new instance of UpdateModelSettingsRequest. </summary>
```

Comment:
use <see cref="UpdateModelSettingsRequest"/>

Reply:
UpdateModelSettingsRequest should be internal and this won't be required. I'll take care of this in this PR or the next